Makefile

```diff
@@ -32,3 +49,40 @@ check:
 .PHONY: build release
 .PHONY: test
 .PHONY: fmt check
+	go build -o ./bin/ezex-users$(EXE) ./cmd/server.go
+
+build_race:
+	go build -race -o ./bin/ezex-users$(EXE) ./cmd/server.go
+########################################
+### Proto
+proto: proto-format
+	rm -rf api/gen
+	cd api && buf generate --template ./proto/buf.gen.yaml --config ./proto/buf.yaml ./proto
```

Comment:
Please avoid using buf. That is more than what we need in this service. Use same approach as [Makefile](https://github.com/ezex-io/ezex-notification/blob/4865f1fe6c6999d60e1ba0441422203257eff58d/Makefile#L15) in notification service.